src/Pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,120 @@
+import { useEffect, useMemo, useState } from 'react';
+import { useSearchParams } from 'react-router-dom';
+import { getPeople } from '../../api';
+import { Loader } from '../../components/Loader';
+import { Person } from '../../types';
+import { PeopleFilters } from '../../components/PeopleFilters';
+import { PeopleTable } from '../../components/PeopleTable';
+import { filtersPeople } from '../../utils/filtersPeople';
+import { sortPeople } from '../../utils/sortPeople';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [loading, setLoading] = useState(false);
+  const [noPeopleMsg, setNoPeopleMsg] = useState(false);
+  const [showTable, setShowTable] = useState(false);
+  const [errorMsg, setErrorMsg] = useState(false);
+
+  const [searchParams] = useSearchParams();
+
+  const query = searchParams.get('query') || '';
+  const sex = searchParams.get('sex') || null;
+  const centuries = searchParams.getAll('centuries') || [];
+  const sort = searchParams.get('sort') || null;
+  const order = searchParams.get('order') || null;
+
+  const filteredPeople = useMemo(() => {
+    return filtersPeople(people, query, centuries, sex);
+  }, [people, query, centuries, sex]);
+
+  const sortedPeople = useMemo(() => {
+    return sortPeople(filteredPeople, order, sort);
+  }, [filteredPeople, sort, order]);
+
+  const visiblePeople = sortedPeople;
```

Comment:
You can directly use 'sortedPeople' in the 'PeopleTable' component instead of assigning it to 'visiblePeople'.